mlflow/projects/__init__.py

```diff
@@ -100,6 +101,7 @@ def _run(
                 backend_config,
                 tracking_store_uri,
                 experiment_id,
+                skip_image_build,
```

Comment:
This is a backwards-incompatible change for third-party project backends that do not define a `skip_image_build` parameter in the `run()` method. Can we use `inspect` to determine whether the argument is present and, if it isn't, print a warning?

Reply:
Good catch. Can we use `backend_config`?